configs/community.dbuild

```diff
@@ -2302,17 +2286,20 @@ build += {
     ]
   }
 
-  // frozen (October 2018) at September 2018 commit before upgrade to
-  // fs2 v1.0.0-RC1 which we can't handle yet
   ${vars.base} {
     name: "sttp"
     uri:  ${vars.uris.sttp-uri}
     extra.sbt-version: ${vars.sbt-1-version}
     // aggregates all JVM projects
     extra.projects: ["rootJVM"]
-    // I got some compile error here (September 2018) that seemed unlikely to be
-    // significant. it's fine, we don't need to have every last subproject
-    extra.exclude: ["monix", "okhttpBackendMonix", "asyncHttpClientBackendMonix"]
+    extra.exclude: [
+      // I got some compile error here (September 2018) that seemed unlikely to be
+      // significant. it's fine, we don't need to have every last subproject
+      "monix", "okhttpBackendMonix", "asyncHttpClientBackendMonix"
+      // doesn't appear to have caught up with fs2-reactive-streams no longer being in a separate repo
+      // (October 2018)
+      "asyncHttpClientBackendFs2"
```

Comment:
It has, since https://github.com/softwaremill/sttp/pull/151 and https://github.com/softwaremill/sttp/pull/153

Reply:
trying at https://github.com/scala/community-builds/pull/837 to re-add it